### PR TITLE
CFP reminder and peripheral fixes

### PR DESCRIPTION
**Pull request by thatdocslady**
Comments/commits welcome. Special note for sponsor welcome - I went by the website, are there new ones to add?

### CODE REVIEW

File: docs/conf/eu/2017/cfp.rst [hunk 1]
```diff
@@ -83,22 +84,22 @@ an anchor session.
...
+*Please note that for unconference sessions we will not be able to waive your attendance fee*.
+We hope to have many good conversations, but can't afford to offer each Unconference session leader a free ticket.
 
-If your Unconference session is selected,
-we won't be offering a free ticket.
-We hope to have many good conversations,
-but can't afford to offer each Unconference session leader a free ticket.
+You’ll hear from us with our proposal decisions by **15th June**.
 
 Questions?
```
Comment (ericholscher) [inline docs/conf/eu/2017/cfp.rst:104]: Just a thought, but we've already released the CFP -- is it cool to change it after people have already submitted? Should we ping them to tell them?

File: docs/conf/eu/2017/news/cfp-reminder.rst [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+Although the deadline is fast approaching, there is still time to submit a talk proposal for Write the Docs Prague! We'll be accepting proposals until **Wednesday, May 31st at Midnight CET**.
+
+Also, if you're interested in pitching a talk, but you need some ideas or guidance, check out our `Call for Proposals <http://www.writethedocs.org/conf/eu/2017/cfp/>`_ page. If you've got an idea, but you're having a hard time getting your proposal to come together, you can always ask for feedback on Slack. Your fellow documentarians are there to help!
+
+We can't wait to see your submissions! We'll be reviewing all the talk proposals in the beginning of June and announcing the full conference schedule by mid-June.
+
+Welcome and thanks to our first 2017 sponsors
+---------------------------------------------
+
+This conference wouldn't happen without the help of our amazing sponsors. So far we're very grateful to have `Wordpress <https://wordpress.com/>`_ and `Knowledge Owl <https://www.knowledgeowl.com/>`_ join us as sponsors for this year's conference.
```
Comment (ericholscher) [inline docs/conf/eu/2017/news/cfp-reminder.rst]: Wordpress.com, not just "Wordpress"

File: docs/conf/eu/2017/news/cfp-reminder.rst [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+We still have lots of exciting opportunities for sponsorship, if your company is looking for ways to support good documentation. Drop us an email at at sponsorship@writethedocs.org with any questions.
+
+Thanks so much for your help in making Write the Docs possible!
+
+More details coming soon
+------------------------
+
+We are currently hammering out the last details for this year's event. Expect to hear more about the boat tour, writing day, unconf, party, reception, and more in the next few weeks. It's always so thrilling for us to see how the conference improves and evolves – this year will be no exception.
```
Comment (ericholscher) [inline docs/conf/eu/2017/news/cfp-reminder.rst]: `unconf` feels too informal.

**Review by ericholscher — COMMENTED** [review]
Looks good, just some small chagnes.

File: docs/conf/eu/2017/news/cfp-reminder.rst [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+   :tags: 2017, prague,cfp
+
+Call for Proposals Reminder
+===========================
+
+Greetings, fellow documentarians!
+
+The conference is now a little over three months away, and we hope you’re getting geared up for another unforgettable Write the Docs experience in beautiful Prague, Czech Republic.
+
+`Tickets <http://www.writethedocs.org/conf/eu/2017/tickets>`_ are selling quickly – thanks to everyone who has already purchased theirs! We do expect the conference to sell out fully again this year, so hop off that fence you might be sitting on!
```
Comment (plaindocs) [inline docs/conf/eu/2017/news/cfp-reminder.rst:13]: PLEASE purchase your ticket first, THEN your flights and accommodation? :imp:

**Comment by plaindocs** [comment]
LGTM

File: docs/conf/eu/2017/news/cfp-reminder.rst (hunk 4 above)
Comment (thatdocslady) [reply docs/conf/eu/2017/news/cfp-reminder.rst:13]: I think that's a bit much ;)

**Comment by thatdocslady** [comment]
Updated with comments, ready for merge if OK

**Pull request merged**